ui/css/treeherder-resultsets.css

```diff
@@ -187,3 +205,19 @@ fieldset[disabled] .btn-resultset:hover {
 .job-list table tr:nth-child(even) {
   background: #f8f8f8;
 }
+
+.get-next {
```

Comment:
Bootstrap 4 does away with the concept of a ``well`` so this adds the styling to get it to look like that again.